.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
always name your branch with your prefix and a meaningful name such as 
iqra/create-react-app